src/solver/types.ts

```diff
@@ -50,7 +50,7 @@ export const BodyMessageSchema = z.object({
   signerPubKey: z.string().optional(),
 });
 
-export const WakuMessageSchema = z.object({
+const WakuMessageSchema = z.object({
```

Comment:
It doesn't hurt us so, let's export it anyway
```suggestion
export const WakuMessageSchema = z.object({
```